apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -65,12 +65,7 @@ function constructRulingPdf(
     }),
   )
   setLineGap(doc, 1)
-  addNormalJustifiedText(
-    doc,
-    formatMessage(ruling.intro, {
-      courtStartDate: formatDate(theCase.courtStartDate, 'PPP'),
-    }),
-  )
+  addNormalJustifiedText(doc, theCase.introduction ?? '')
```

Comment:
As this is a required field, consider defaulting to some CF text such as "Aðfararorð hafa ekki verið skráð".

Reply:
Also consider deleting the CF string `ruling.intro`. It has not been modified in CF and should be safe to delete.

Reply:
Ruling.intro removed.

I'm confused about defaulting to a CF text. Why would we default to something when the string is required? That means that there will always be a value and no need for a default string, right?